ksql-serde/src/main/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroDeserializer.java

```diff
@@ -75,8 +75,14 @@ public GenericRow deserialize(final String topic, final byte[] bytes) {
       Map<String, String> caseInsensitiveFieldNameMap = getCaseInsensitiveFieldMap(genericRecord);
       List columns = new ArrayList();
       for (Field field : schema.fields()) {
-        columns.add(enforceFieldType(field.schema(), genericRecord
-            .get(caseInsensitiveFieldNameMap.get(field.name().toUpperCase()))));
+        // Set the missing fields to null. We can make this configurable later.
```

Comment:
When will this be true? Wouldn't it mean there was an error with the schema conversion?

Reply:
This can happen anytime when KSQL schema is different than the avro schema. If user defines a column that does not exist in the avro schema or if the avro schema changed.

Reply:
Should this still be an error? Or at least some logging? At the moment it is going to be silently ignored